AI/modules/signal/models/transformer/wrapper.py

```
@@ -0,0 +1,113 @@
+            verbose=verbose,
+            **kwargs 
+        )
+        return history
+
+
+
+
+    def predict(self, X_input: np.ndarray) -> np.ndarray:
+        """추론 수행"""
+        if self.model is None:
+            raise ValueError("모델이 없습니다. load()하거나 build() 하세요.")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)